### ISSUE TITLE
Multitracker support

### ISSUE THREAD

**Issue by dxpsite** [taobataoma/meanTorrent#47]
Good day! Can add multitracker torrents support?

**Comment by taobataoma** [comment]
@dxpsite hi, can you give me more details? has any wrong on meanTorrent now?

**Comment by dxpsite** [comment]
Multitracker is an extension of the BitTorrent metadata format, proposed by John Hoffman, author of BitTornado. It is in the addition of the dictionary "announce-list" (instead of the "announce" field), which allows to specify not one, but several trackers. Support completely depends on the client, does not require changes in trackers, does not carry protocol changes. In the event of the failure of one of the trackers, the extension allows you to continue the file-sharing process.

**Comment by taobataoma** [comment]
@dxpsite I know the protocol with announce-list, but I want know where need changed in meanTorrent. because you can put more than one tracker when you create a torrent file. you want to list all trackers in announce-list on torrent detail page of meanTorrent? or other?

**Comment by dxpsite** [comment]
The list is optional) The main thing is that there should be a multitracker

2018-02-01 11:50 GMT+03:00 taobataoma <notifications@github.com>:

**Comment by taobataoma** [comment]
@dxpsite I need to know the specific places that need to be modified in meanTorrent. 

meanTorrent is a private tracker cms. your mean is torrents file need include 'announce-list' field? or meanTorrent need to create more than one tracker server? usual torrents file of private tracker only need one tracker url in it, is it right?

**Comment by dxpsite** [comment]
Right. Many torrent trackers in Russia are banned and we bypass the ban on
VPN and distribute in our closed tracker because we are adding additional
Russian subtitles. It's comfortable. In addition, we also place our
releases with hardcoded subtitles.

2018-02-01 12:21 GMT+03:00 taobataoma <notifications@github.com>:

**Comment by taobataoma** [comment]
@dxpsite I write two question above, but you said 'right', witch one is your answer? 

I am a  chinese, my english is very poor, so I am difficult to understand your questions. many torrent trackers in china are banned also. I still don't understand how I should optimize meanTorrent. ;(

**Comment by dxpsite** [comment]
Okay, I want to list all trackers in announce-list on torrent detail page

**Comment by taobataoma** [comment]
@dxpsite Ok, I`ll support it soon.
